composer.json

```diff
@@ -82,7 +82,9 @@
         "doctrine/dbal": "^2.6",
         "filp/whoops": "^2.4",
         "guzzlehttp/guzzle": "^6.5.5|^7.0.1",
-        "league/flysystem-cached-adapter": "^1.0",
+        "league/flysystem-aws-s3-v3": "^2.0",
+        "league/flysystem-ftp": "^2.0.0-alpha.4|^2.0",
```

Comment:
`^2.0.0-alpha4` is enough, the `|^2.0` is included in that range and redundant.

Reply:
Well, technically ^2.0 includes even more versions, as I just tweeted, so actually it needs to be gone otherwise it'll still allow alpha1

Reply:
Right. Good catch 👍 

I think ultimately I was trying to find a way to keep the "^2.0" constraint so the PR would be "ready to go" as soon as Flysystem v2 got released but I guess that's not possible as long as it's unreleased. Anyway, thanks for helping out!

Reply:
It is ready to go, `^2.0-alpha4` is completely valid actually more accurate and correct than ^2.0 (which again resolves to `^2.0-dev` internally). It just looks odd because we don't see a lot of these, but it's not wrong.

Reply:
That's true. Hadn't thought of it like that yet.